general/toc.yml

```diff
@@ -17,12 +17,14 @@
 
 - name: Networking
   items:
-  - uid: network-reverse-proxy
-  - uid: network-reverse-proxy-apache
-  - uid: network-reverse-proxy-caddy
-  - uid: network-reverse-proxy-haproxy
-  - uid: network-reverse-proxy-nginx
-  - uid: network-reverse-proxy-traefik
+  - uid: network-main
+  - name: Reverse Proxies
```

Comment:
The name should probably just be set in the file rather than here.

Reply:
So I was trying to mimic the media section if you want to compare them. I'm open to suggestions, wasn't too sure how I wanted to organize em

Reply:
Ah I see the purpose, this can work here. I would replace `- name: Networking` with `- uid: network-index` though. We want to leave index pages on the titles themselves as much as possible.